Gemfile.lock

```diff
@@ -173,11 +176,13 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  bcrypt (~> 3.1.7)
```

Comment:
will we need bcrypt if the front-end will be doing OAuth

Reply:
Might as well leave it for now and can adjust later. Looks good!